documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -1,3 +1,17 @@
+= Upgrading from 13.0 to 14.0
+
+== Cross Site Replication
+
+=== Asynchronous backup strategy logging
+
+By default,  asynchronous backup strategy starts to logging exceptions received from remote sites.
+To go back to the old behaviour, set the `failure-policy="IGNORE"`, like:
```

Comment:
```suggestion
By default, the asynchronous backup strategy logs exceptions that occur on remote sites.
To disable exceptions from backup locations and log exceptions for the local site only, set `failure-policy="IGNORE"` in your configuration as follows:
```

Reply:
@oraNod actually, `and log exceptions for the local site only` do not make much sense since we don't cross-site replicate to itself.

Reply:
@pruivo That makes sense. I made some edits to the suggestion.